### DISCUSSION TITLE
Zigbee Binding

### DISCUSSION THREAD

**Discussion by dbroome1978** [Koenkk/zigbee2mqtt#21937] [Devices]
I have started playing with this feature while the ember until the ember driver becomes stable on my install (and wow - there has been a lot of progress on this.)

From what I am seeing based on what binding works, both devices need to have an extra cluster to be able to set the bind and they need to have onoff on that cluster, So even if the swtich supports binding (my Nue / 3A switches do and have succesfully binded to my gridwel led strips), if the light itself doesn't have an extra cluster with the onoff, I can't bind it (for example - all my sengled bulbs.) 

Firstly - is my understandinig correct?

Secondly - if so, is there any way to see on the device compatiblity page which devices support this? I tried comparing [my led strips ](https://www.zigbee2mqtt.io/devices/GD-ZCRGB012.html) to [my light bulb](https://www.zigbee2mqtt.io/devices/E11-G13.html) to find the difference but don't see this field listed

**Comment by deviantintegral** [comment]
I'm not sure about needing the extra cluster for direct binding.

However, you could probably work around that by binding to a group instead of a device. In that case, the remote sends a broadcast to the group, and as long as the target device supports being joined to the group it should respond.

I've had trouble with Sengled bulbs myself. The colour-temperature bulbs I bought had really poor life, and would die by flickering rapidly between cool and warm. As well, the colour bulbs I have seem to have odd issues with binding: https://github.com/Koenkk/zigbee2mqtt/discussions/11543

I've stopped buying their bulbs in favour of Hue. I will say the Sengled power switches I have are rock solid.

I don't think binding is part of the docs, as there's many odd implementations out there on the remote side too.

**Comment by avbdr** [comment]
im sorry to hijack a thread, but were somebody able to bind switch to the switch? I want to implement a 3way switch using zb groups/bindings, but it seems that its not something which  work out out of the box.

**Reply by deviantintegral** [reply]
Oh, I can answer that! I have two of the above Enbrighten switches bound to each other as devices. It works great - basically turns my two different outside lights on different switches into one single light (even if Home Assistant is down).

I also have an Enbrighten dimmer directly bound to a Sengled plug. Again, it works perfectly - on / off on the wall turns on the overhead dining room light, as well as the lights in the china cabinet. And dimming the overhead by pressing and holding it doesn’t affect the switch (since the plug only accepts an on / off cluster, and not a level control cluster). If I want individual control, I can do it from Home Assistant.

https://www.zigbee2mqtt.io/devices/43076.html#enbrighten-43076
https://www.zigbee2mqtt.io/devices/E1C-NB7.html#sengled-e1c-nb7

**Comment by avbdr** [comment]
@deviantintegral yeh, Im aware that some devices do it out of the box. I were wondering if there is something can be done with devices which doesnt support direct binding. 
In the doc z2m were mentioning that you can use groups for that, but seems this approach works only with lights. 
Looking for a way not to change my switches :)

**Comment by dbroome1978** [comment]
@deviantintegral It seems to be hit or miss on using groups. It gives me the options when I set up - for example - plugs as a group, but then it doesn't actually control the group when i use the switch.
